bikeshed/MetadataManager.py

```diff
@@ -744,6 +745,7 @@ def joinDdList(a,b):
     "Repository": Metadata("Repository", "repository", joinValue, parseRepository),
     "Revision": Metadata("Revision", "level", joinValue, parseLevel),
     "Shortname": Metadata("Shortname", "shortname", joinValue, parseLiteral),
+    "Show Hide Deleted Button": Metadata("Show 'Hide deleted text' button", "showHideDeletedButton", joinValue, parseBoolean),
```

Comment:
The first arg is what you actually write in your metadata; is this what you really want people to write?